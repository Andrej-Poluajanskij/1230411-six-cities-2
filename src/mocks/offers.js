const offers = [
  {
    image: `img/apartment-01.jpg`,
    price: `120€`,
    descriptions: `Beautiful & luxurious apartment at great location`,
    type: `Apartment`,
    coordinates: [52.3909553943508, 4.85309666406198]
  },
  {
    image: `img/room.jpg`,
    price: `80€`,
    descriptions: `Wood and stone place`,
    type: `Private room`,
    coordinates: [52.369553943508, 4.85309666406198]
  },
  {
    image: `img/apartment-02.jpg`,
    price: `132€`,
    descriptions: `Canal View Prinsengracht`,
    type: `Apartment`,
    coordinates: [52.3909553943508, 4.929309666406198]
  },
  {
    image: `img/apartment-03.jpg`,
    price: `180€`,
    descriptions: `Nice, cozy, warm big bed apartment`,
    type: `Apartment`,
    coordinates: [52.3809553943508, 4.939309666406198]
  },
];

export default offers;
