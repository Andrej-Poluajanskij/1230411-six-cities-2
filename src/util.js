let cityMapContainer = () => {
  const div = global.document.createElement(`div`);
  div.setAttribute(`id`, `map`);
  global.document.body.appendChild(div);
  return div;
};

export default cityMapContainer;
