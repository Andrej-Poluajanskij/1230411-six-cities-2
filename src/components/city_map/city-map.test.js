import React from 'react';
import render from 'react-test-renderer';
import Map from './city-map';

import offers from '../../mocks/offers';
import util from '../../util';

it(`CityMap correctly renders after relaunch`, () => {
  util();

  const tree = render
  .create(<Map
    mapMarks={offers}
  />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});


