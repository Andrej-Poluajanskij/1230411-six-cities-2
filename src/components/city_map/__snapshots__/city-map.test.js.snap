// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CityMap correctly renders after relaunch 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "97%",
      "marginTop": "14px",
    }
  }
/>
`;
